docs/source/metrics.rst

```diff
@@ -196,12 +196,71 @@ Metric API
 .. autoclass:: pytorch_lightning.metrics.Metric
     :noindex:
 
-*************
-Class metrics
-*************
+***************************
+Class vs Functional Metrics
+***************************
 
+The functional metrics follow the simple paradigm input in, output out. This means, they don't provide any advanced mechanisms for syncing across DDP nodes or aggregation over batches. They simply compute the metric value based on the given inputs.
+
+Also the integration within other parts of PyTorch Lightning will never be as tight as with the class-based interface.
```

Comment:
Might want to re-word this

Reply:
Sure, have any ideas how? I did not actually write this - this is taken from the current documentation (if it were up to me it would be taken out entirely)

Reply:
@teddykoker do you have any suggestions?

Reply:
Hmm my bad, I'm not opposed to removing either, I think this just leads to more confusion